app/src/androidTest/java/com/example/partyPlayer/ExampleInstrumentedTest.kt

```diff
@@ -1,4 +1,4 @@
-package com.example.party_player
+package com.example.partyPlayer
```

Comment:
Changed the name of the package to not include underscores in accordance with naming conventions here:
https://kotlinlang.org/docs/reference/coding-conventions.html#naming-rules

Reply:
Actually, I'd take another option... all lowercase is typical for packages.  Kotlin discourages multiple words.
"Names of packages are always lower case and do not use underscores (org.example.project). Using multi-word names is generally discouraged, but if you do need to use multiple words, you can either simply concatenate them together or use the camel case (org.example.myProject)."